<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthDialog</class>
 <widget class="QDialog" name="AuthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelLogin">
     <property name="text">
      <string>Логин:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtLogin">
     <property name="placeholderText">
      <string>Введите логин</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelPassword">
     <property name="text">
      <string>Пароль:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtPassword">
     <property name="placeholderText">
      <string>Введите пароль</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnLogin">
       <property name="text">
        <string>Войти</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
